src/integrationTest/java/uk/gov/hmcts/darts/authentication/controller/impl/LoginOrRefreshIntTest.java

```diff
@@ -20,15 +20,13 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @WebMvcTest
+@ComponentScan({"uk.gov.hmcts.darts"})
```

Comment:
As this test effectively needs multiple component layers (beyond the web layer), I think it would be more idomatic to perfom the full component scan by replacing these annotations with the following:
```
@SpringBootTest
@AutoConfigureMockMvc
@ActiveProfiles({"intTest", "h2db"})
```

Reply:
I've tried this, but the integrationTest fails for this WebMvcTest - it looks like we need `ComponentScan` for it to work

Reply:
Done in commit e3591a1